src/index.ts

```diff
@@ -90,8 +95,12 @@ export default class HDKeyring implements Keyring<SerializedHDKeyring> {
 
     this.#mnemonic = mnemonic
 
+    const passphrase = hdOptions.passphrase || ""
+
+    this.#passphrase = passphrase
+
     this.path = hdOptions.path
-    this.#hdNode = HDNode.fromMnemonic(mnemonic, undefined, "en").derivePath(
+    this.#hdNode = HDNode.fromMnemonic(mnemonic, passphrase, "en").derivePath(
```

Comment:
Above, we default `passphrase` to `""`. Prior, we passed `undefined`. Are we sure the underlying outcome in `HDNode` is identical for those two parameter values?

Reply:
Yes.
https://github.com/beemeeupnow/ethers.js/blob/73d67daf9ba41d7eb71e8c7b1a0dbd8e2de4ef56/packages/hdnode/src.ts/index.ts#L314